{
  "public_data_at_usn": "0001-01-01T00:00:00Z",
  "crd": "0001-01-01T00:00:00Z",
  "candidate": "CVE-2021-4024",
  "public_date": "2021-12-23T20:15:00Z",
  "references": [
    {
      "reference": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-4024"
    },
    {
      "reference": "https://bugzilla.redhat.com/show_bug.cgi?id=2026675"
    },
    {
      "reference": "https://twitter.com/discordianfish/status/1463462371675066371"
    },
    {
      "reference": "https://github.com/containers/podman/pull/12283"
    }
  ],
  "description": "A flaw was found in podman. The `podman machine` function (used to create and manage Podman virtual machine containing a Podman process) spawns a `gvproxy` process on the host system. The `gvproxy` API is accessible on port 7777 on all IP addresses on the host. If that port is open on the host's firewall, an attacker can potentially use the `gvproxy` API to forward ports on the host to ports in the VM, making private services on the VM accessible to the network. This issue could be also used to interrupt the host's services by forwarding all ports to the VM.",
  "ubuntu_description": "",
  "notes": [],
  "bugs": [
    {
      "bug": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1000844"
    }
  ],
  "priority": "medium",
  "discovered_by": "",
  "assigned_to": "",
  "patches": [
    {
      "package_name": "libpod",
      "release_patches": [
        {
          "release_name": "trusty",
          "status": "ignored",
          "note": "out of standard support"
        }
      ]
    }
  ],
  "upstreams": []
}
